.github/workflows/collector-full.yml

```diff
@@ -0,0 +1,141 @@
+name: Build the collector full image
+
+on:
+  workflow_call:
+    inputs:
+      collector-tag:
+        type: string
+        required: true
+        description: |
+          The tag used to build the collector image, usually extracted from the
+          collector-slim.yml workflow
+      skip-built-drivers:
+        type: boolean
+        required: true
+        description: |
+          Skip built drivers if no new drivers were built.
+
+jobs:
+  build-collector-full:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'build-full-images') }}
```

Comment:
I decided to put this in a separate workflow because it will always run either `build-collector-full` or `retag-collector-full`, think this way it is easier to keep track of this detail and it also should show them in the graph as being together, WDYT?

Reply:
yeah sounds reasonable!